packages/response-ops/rule_form/src/rule_actions/rule_actions_item.tsx

```diff
@@ -692,7 +691,7 @@ export const RuleActionsItem = (props: RuleActionsItemProps) => {
       }
       buttonContentClassName="eui-fullWidth"
       buttonContent={
-        <EuiPanel color="subdued" paddingSize="m">
+        <EuiPanel color={subdued} paddingSize="m">
```

Comment:
Question: Why do we need to override the default `subdued` panel here? 🤔

Reply:
this is still WIP, i ll update it